src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java

```diff
@@ -71,6 +71,10 @@ private void setProcessAudioVideoActionDropdown(SscsCaseData sscsCaseData, boole
             populateListWithItems(listOptions, EXCLUDE_EVIDENCE);
         }
 
+        if (!hasJudgeRole || hasSuperUserRole) {
+            populateListWithItems(listOptions, SEND_TO_JUDGE);
```

Comment:
Shouldn't TCW be able to trigger sendToJudge?

Reply:
yes, hence they are not judges

Reply:
Do you even need the 'hasSuperUserRole' part @ryandac ? Isn't it implied that if you are not a judge, then you will always have this.

Reply:
superusers need to have all options. I wasn't sure if a superuser would also have a judge role, which is why I explicitly add the condition

Reply:
ok fair enough. In reality they should never have both but i guess better to be safe.

Reply:
I didn't know, good to know that is the case.